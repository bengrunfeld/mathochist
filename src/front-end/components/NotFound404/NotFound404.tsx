import React from 'react'

const NotFound404 = () => (
  <div> 
    <h1>404 - Not found...</h1>
  </div>
)

export default NotFound404