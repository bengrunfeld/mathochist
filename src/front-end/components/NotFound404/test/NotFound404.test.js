import React from 'react'
import Enzyme, { shallow, mount, render } from 'enzyme'
import { MemoryRouter } from 'react-router-dom'

import NotFound404 from '../NotFound404'


describe('<NotFound404 />', () => {
  test('matches the snapshot', () => {
    const tree = mount(<NotFound404 />)
    expect(tree).toMatchSnapshot()
  })
})
