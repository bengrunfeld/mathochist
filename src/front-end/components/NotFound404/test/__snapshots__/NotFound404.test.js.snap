// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound404 /> matches the snapshot 1`] = `
<NotFound404>
  <div>
    <h1>
      404 - Not found...
    </h1>
  </div>
</NotFound404>
`;
